### ISSUE TITLE
Question on SBM-Cluster

### ISSUE THREAD

**Issue by yyou1996** [graphdeeplearning/benchmarking-gnns#9]
Hi,

Very grateful for sharing the project! I have a question on SBM-Cluster. It is a semi-supervised node classification task, and in paper it is said to use one single label for each community (for training?), but in code it seems that for each training graph all labels are used. Did I misunderstand something?

**Comment by chaitjo** [comment]
Hi @yyou1996, thanks for your interest. I believe SBM are implemented exactly as described in the paper. If not, we'd like to investigate further. 

Could you point to the exact sections of the code where you feel there's a discrepancy?

**Comment by yyou1996** [comment]
Hi @chaitjo, the lines 27 and 28 in benchmarking-gnns/train/train_SBMs_node_classification.py, I did not see the "semi-supervised" setting: it seems that all the labels in graph are used to train, but in paper it is said to use one single label for each community (for training?).
`batch_scores = model.forward(batch_graphs, batch_x, batch_e, batch_snorm_n, batch_snorm_e)
loss = model.loss(batch_scores, batch_labels)`

I think I must miss something. Thank you very much for your reply!

**Comment by chaitjo** [comment]
Please look at the file for how the dataset SBMs is created: https://github.com/graphdeeplearning/benchmarking-gnns/blob/master/data/SBMs/generate_SBM_CLUSTER.ipynb

If I understand your question, is the issue that `batch_labels` contains labels for each node in the graph?

**Comment by yyou1996** [comment]
Very grateful for your answer, it solves my confusion.
